FitOn/settings.py

```diff
@@ -1,19 +1,68 @@
 import os
 from pathlib import Path
 from datetime import timedelta
+import boto3
+import os
```

Comment:
os is imported twice, it's already imported above

Reply:
Resolved